common/test/src/View/GroupNodesTest.cpp

```diff
@@ -154,6 +154,45 @@ namespace TrenchBroom {
             return false;
         }
 
+        TEST_CASE_METHOD(GroupNodesTest, "GroupNodesTest.copyPasteGroupResetsDuplicateGroupId", "[GroupNodesTest]") {
+            auto* entityNode = new Model::EntityNode{};
+            document->addNodes({{document->parentForNodes(), {entityNode}}});
```

Comment:
There should be a convenience function for this

Reply:
There is, but I actually prefer this notation, which is one reason why I had removed the overloads in the first place.